selfdrive/ui/ui.h

```diff
@@ -78,19 +79,22 @@ struct Alert {
       // Handle controls timeout
       if (controls_frame < started_frame) {
         // car is started, but controlsState hasn't been seen at all
-        alert = {"openpilot Unavailable", "Waiting for controls to start",
+        alert = {qApp->translate("Alert", QT_TR_NOOP("openpilot Unavailable")),
+                 qApp->translate("Alert", QT_TR_NOOP("Waiting for controls to start")),
```

Comment:
Do we need "Alert" at all? It can figure that out if left without

Reply:
it's not inherit from QObject. 
Closed for now. It might be better to move alerts to class OnroadAlert, translate these messages together with events  from controls/events.py.

Reply:
think this one is better: https://github.com/commaai/openpilot/pull/32024